batch-1/movie-info/akshay/filminfo.rb

```diff
@@ -0,0 +1,88 @@
+require 'open-uri'
+require 'json'
+
+
+
+def input
+	puts"Enter the Movie title:"
+	movie_name=gets.chomp
+	movie_name=movie_name.split(" ").join("+")
+	return movie_name
+end
+
+def generate_page movie_name
```

Comment:
It's a good convention to write parenthesis in function definition lines, in your case ```generate_page(movie_name)```.